src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int BALL_SUPPLIER = 100;
```

Comment:
```suggestion
    public static final int MAX_NUMBER = 100;
```
lets choose better naming